app/src/main/java/org/dhis2/App.java

```diff
@@ -132,9 +132,6 @@ public void onCreate() {
             upgradeSecurityProviderSync();
 
         setUpAppComponent();
-        if (wantToImportDB) {
```

Comment:
maybe you could remove `wantToImportDB` property and `populateDBIfNeeded` method

Reply:
Thanks !! totally forgot